### PR TITLE
[IMP] subscriptions: corrected error about recurring products tip

### PR DESCRIPTION
**Pull request by ksc-odoo**
Fixed error re: tip on the products.rst of Subscriptions to adhere to suggestion in this PROJECT TASK: https://www.odoo.com/web#id=3744972&cids=3&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/sales/subscriptions/products.rst [hunk 1]
```diff
@@ -71,6 +71,10 @@ Doing so reveals a blank product form, which can be configured and customized in
    :align: center
    :alt: A basic subscription product form in Odoo Subscriptions application.
 
+.. tip::
+   The :guilabel:`Recurring` option on a product form should be unchecked in order to sell it
+   without a recurrence.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/products.rst:76]: Don't think we need this?
```suggestion
```

File: content/applications/sales/subscriptions/products.rst [hunk 2]
```diff
@@ -101,8 +105,7 @@ that recurrence period.
    An existing product can be made into a subscription product, simply by marking it as
    :guilabel:`Recurring`, and configuring :guilabel:`Time-based pricing` on the product form.
 
-   A subscription product can still be sold as a regular product, by adding it to a quotation and
-   *not* selecting a :guilabel:`Recurrence` on the quotation.
+   A recurrence is **required** for recurring products.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/products.rst:108]: Don't think we need this either, honestly?
```suggestion
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 
Couple comments for you below.
I'm not sure we need either of these content changes— can just simply remove what's there and that solves the issues outlined in the Task. Wdyt?

**Comment by ksc-odoo** [comment]
Made all your suggested adjustments @StraubCreative -- ready for another quick look from you whenever you get a chance. Thanks! 👍

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**